packages/pipeline/src/utils/get_ohlcv_trading_pairs.ts

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import { chain } from 'ramda';
+import { Connection } from 'typeorm';
+
+export interface TradingPair {
+    fromSymbol: string;
+    toSymbol: string;
+    latest: number;
+}
+
+interface StaticPair {
+    fromSymbol: string;
+    toSymbol: string;
+}
+
+const COINLIST_API = 'https://min-api.cryptocompare.com/data/all/coinlist?BuiltOn=7605';
+
+interface CryptoCompareCoinListResp {
+    Data: Map<string, CryptoCompareCoin>;
+}
+
+interface CryptoCompareCoin {
+    Symbol: string;
+    BuiltOn: string;
+    SmartContractAddress: string;
+}
+
+const TO_CURRENCIES = ['USD', 'EUR', 'ETH'];
```

Comment:
Can you include USDT here? So much token / token volume happens on binance and i think USDT is critical to look at.

Reply:
We should maybe turn this into a param at some point but for now I will just add USDT.